src/compiler/checker.ts

```diff
@@ -18864,28 +18862,34 @@ namespace ts {
                 }
             }
 
-            function getDeclarationSpaces(d: Declaration): SymbolFlags {
+            const enum DeclarationSpaces {
+                None = 0,
+                ExportValue = 1 << 0,
+                ExportType = 1 << 1,
+                ExportNamespace = 1 << 2,
+            }
+            function getDeclarationSpaces(d: Declaration): DeclarationSpaces {

```

Comment:
This is internal to `checkExportsOnMergedDeclarations`. It was just re-using the `SymbolFlags` enum, but only using 3 of the flags -- so I just gave it its own enum.